metabase/metabase
# Inline Category Picker

## Background

We want to make it super easy to filter by categories when there are less than 13 unique category values. when opening the bulk filter modal, we will show inline checkboxes for category filters when there are 12 or fewer (like product category), and use the standard popover filter for larger sets of values (like state).

![Screen Shot 2022-06-14 at 1 54 01 PM](https://user-images.githubusercontent.com/30528226/173676890-8c21df90-d26f-4dcc-90c3-8d8e5dc9bc5b.png)


## To Test

- Open the bulk filter modal on the People table: see that the source field shows its filter options inline
- Also in the people table, see that the state field does not show its options inline, because there are too many
- See that adding and removing category filters for both fields works as expected.
